modules/solr/build.gradle

```diff
@@ -0,0 +1,8 @@
+description = "Testcontainers :: Solr"
+
+dependencies {
+    compile project(':testcontainers')
+    compile 'org.apache.httpcomponents:httpclient:4.5.10'
```

Comment:
FYI to avoid an additional dependency, you can also use OkHttp that comes as a transitive dependency from the core module

Reply:
Should be fixed with 681a5e9caedd17a80fd61a03f0acfa1a7eaa0906